roles/cobbler/defaults/main.yml

```diff
@@ -4,6 +4,21 @@ iso_dir: /var/lib/cobbler/isos
 # Mount point to use for ISOs during import
 iso_mount: /mnt/iso
 
+etc_files:
+ - users.digest
+
+settings:
```

Comment:
The actual setting could be the key here instead of 'name' and 'value'.  Then you could do something like ``settings.signature_url`` in usage.